### ISSUE TITLE
Update BGS Gem for Get Regional Office by Zip Code

### ISSUE THREAD

**Issue by kathleencrawford** [department-of-veterans-affairs/va.gov-team#12934]
## Goal
We need to call the BGS (Regional Office) RO Routing Service to determine the location id based on the Veteran's postal code so that, presumably, the person looking at it is closest to the Veteran.

## Tasks
- [x] Update BGS gem to allow the `getRegionalOfficeByZipCode` data to be utilized

## Acceptance Criteria
- [x] BGS gem code is submitted and a PR is requested for merging

**Comment by kathleencrawford** [comment]
Draft PR submitted here:  https://github.com/department-of-veterans-affairs/bgs-ext/pull/36

It is still a draft because I am waiting on confirmation that I can publish a URL that we are using to access the WSDL.

**Comment by kathleencrawford** [comment]
PR has been approved and merged into master.
